src/tools/options.ts

```diff
@@ -41,6 +46,14 @@ export const SelectionToTemplateParamsMap: Partial<Record<
   StyleLibraryChoice | StateLibraryChoice | ReactNavigationExampleChoice,
   Partial<TemplateParams>
 >> = {
+  [StyleLibraryChoice.GluestackUIDefault]: {
+    hasGluestackUI: true,
+    hasGluestackUIDefault: true
```

Comment:
@hristo87 Lets make it clear that it is the default theme config
```suggestion
    hasGluestackUIDefaultTheme: true
```

Reply:
Done / Tested